docs/development/setting-up.md

```diff
@@ -10,10 +10,11 @@ Swagger UI includes a development server that provides hot module reloading and
 ### Steps
 
 1. `git clone https://github.com/swagger-api/swagger-ui.git`
-2. `cd swagger-ui`
-3. `npm run dev`
-4. Wait a bit
-5. Open http://localhost:3200/
+1. `cd swagger-ui`
+1. `npm install`
+1. `npm run dev`
+1. Wait a bit
+1. Open http://localhost:3200/
```

Comment:
I changed this to all `1.` so future diffs easier, Markdown should already render this correctly, see https://github.com/swagger-api/swagger-ui/blob/cec33bbe2974f336af968a7cef96fd078a06458f/docs/development/setting-up.md. But I can revert to numbers if needed.